[2024-12-03T14:04:32.359+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-03T14:04:32.391+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb.load_data manual__2024-12-03T13:57:32.192678+00:00 [queued]>
[2024-12-03T14:04:32.405+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data_pipeline_mongodb.load_data manual__2024-12-03T13:57:32.192678+00:00 [queued]>
[2024-12-03T14:04:32.406+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-03T14:04:32.432+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_data> on 2024-12-03 13:57:32.192678+00:00
[2024-12-03T14:04:32.463+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=245) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-03T14:04:32.475+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_data_pipeline_mongodb', 'load_data', 'manual__2024-12-03T13:57:32.192678+00:00', '--job-id', '180', '--raw', '--subdir', 'DAGS_FOLDER/etl_data_pipeline_mongodb.py', '--cfg-path', '/tmp/tmp3ghwkgys']
[2024-12-03T14:04:32.478+0000] {standard_task_runner.py:72} INFO - Started process 259 to run task
[2024-12-03T14:04:32.478+0000] {standard_task_runner.py:105} INFO - Job 180: Subtask load_data
[2024-12-03T14:04:33.550+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_data_pipeline_mongodb.load_data manual__2024-12-03T13:57:32.192678+00:00 [running]> on host ed9532b68cd8
[2024-12-03T14:04:33.659+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data_pipeline_mongodb' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-03T13:57:32.192678+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-03T13:57:32.192678+00:00'
[2024-12-03T14:04:33.660+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-03T14:04:33.661+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-03T14:04:33.662+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2024-12-03 14:04:32.392757+00:00
[2024-12-03T14:04:33.662+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb and current dag run status:running
[2024-12-03T14:04:33.663+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-03T14:04:33.839+0000] {logging_mixin.py:190} INFO - Données chargées dans MongoDB, collection 'cleaned_data'
[2024-12-03T14:04:33.841+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-03T14:04:33.856+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-03T14:04:33.857+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_data_pipeline_mongodb, task_id=load_data, run_id=manual__2024-12-03T13:57:32.192678+00:00, execution_date=20241203T135732, start_date=20241203T140432, end_date=20241203T140433
[2024-12-03T14:04:33.878+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-03T14:04:33.878+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-03T14:04:33.879+0000] {logging_mixin.py:190} INFO - Dag name:etl_data_pipeline_mongodb queued_at:2024-12-03 13:57:32.424376+00:00
[2024-12-03T14:04:33.880+0000] {logging_mixin.py:190} INFO - Task hostname:ed9532b68cd8 operator:PythonOperator
[2024-12-03T14:04:33.918+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-03T14:04:33.932+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
